src/config.c

```diff
@@ -274,6 +275,9 @@ int load_config(struct vpn_config *cfg, const char *filename)
 				continue;
 			}
 			cfg->otp_delay = otp_delay;
+		} else if (strcmp(key, "cookie") == 0) {
```

Comment:
Do we want to be able to read cookies from a config file?

I may be wrong, but this doesn't sound very useful.

Reply:
You are right, I put it there "just because", but better skip it.